content/instructor/index.mdx

```diff
@@ -28,6 +28,7 @@ Here are some quick links to articles of interest:
 * [Workshop Production Process](/instructor/workshops)
 * [The egghead Contributor Agreement (contract)](/instructor/legal)
 * [egghead Content Strategy](/instructor/content-strategy)
+* [Quick Resources](/instructor/quick-resources)
```

Comment:
This link might break when #134 is merged because the title of the article changed to.

`badass screencast recording cheatsheet`

Reply:
I'd suggest removing this link for this PR and adding it to #134 
```suggestion
```